### ISSUE TITLE
Warnings with Elixir 1.5.2

### ISSUE THREAD

**Issue by quatermain** [xou/elixlsx#56]
Hello, 

I got couple of warnings with Elixir 1.5.2 and version 0.3.0:
```
warning: String.to_char_list/1 is deprecated, use String.to_charlist/1
  lib/elixlsx/writer.ex:112

warning: String.to_char_list/1 is deprecated, use String.to_charlist/1
  lib/elixlsx/util.ex:56

warning: String.to_char_list/1 is deprecated, use String.to_charlist/1
  lib/elixlsx/util.ex:63

warning: the char_list() type is deprecated, use charlist()
  lib/elixlsx/writer.ex:9

warning: defp style_to_xml_entry/2 is private, @doc's are always discarded for private functions/macros/types
  lib/elixlsx/xml_templates.ex:369

warning: defp make_style_alignment/1 is private, @doc's are always discarded for private functions/macros/types
  lib/elixlsx/xml_templates.ex:440

warning: defp make_cellxfs/2 is private, @doc's are always discarded for private functions/macros/types
  lib/elixlsx/xml_templates.ex:457
```

**Comment by xou** [comment]
Thanks for the report! #57 addresses this but will need to be split up.

**Comment by xou** [comment]
This should now be fixed in version 0.4.
